bedita-app/controllers/api_base_controller.php

```diff
@@ -726,6 +726,24 @@ public function childrenFilter($conditions = null) {
     }
 
     /**
+     * Setup locale settings
+     * override FrontendController::setupLocale
+     * override AppController::setupLocale
+     *
+     * @see bedita-app/controllers/FrontendController#setupLocale()
+     * @see bedita-app/AppController#setupLocale()
+     */
+    protected function setupLocale() {
+        if (!empty($this->currLang)) {
```

Comment:
There's a flow problem:
 * `setupLocale()` is invoked first from `AppController::beforeFilter()` when `$this->currLang` is empty
 * `$this->currLang` is set after in `getObjects()` with no config check and write

Also setting `currLang` in view is probably not needed